lib/Core/StatsTracker.cpp

```diff
@@ -259,6 +267,10 @@ StatsTracker::~StatsTracker() {
 }
 
 void StatsTracker::done() {
+  // Update statistics for reachability analysis
+  if (updateMinDistToUncovered && (statsFile || OutputIStats))
+      computeReachableUncovered();
```

Comment:
We need to look at where MinDistToUncovered is written.  It looks like only in istats, so we should guard this accordingly